languages/csharp/AppEncryption/AppEncryption/Persistence/AdoMetastorePersistenceImpl.cs

```diff
@@ -23,12 +23,10 @@ public class AdoMetastorePersistenceImpl : IMetastorePersistence<JObject>
 
         private const string LoadQuery = @"SELECT key_record from encryption_key where id = @id and created = @created";
 
-        // Store only if a value with the same key doesn't exist
-        // Using this approach since dotnet doesn't provide a specific integrity violation exception
+        // We try to store even if a value with the same key exists
```

Comment:
we can probably omit the comment entirely since we're no longer doing anything special with the insert statement